packages/aws-cdk/test/integ/cli.exclusions.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
TBH feels like redundant future proofing, but I am fine with it.

Reply:
Its one of those things that will get super hacky and ugly if we don't come up with something half-reasonable now.

Reply:
Why? We can always add this capability when we really need it. There is no added value in baking it to releases that don’t need it.

But still this is fine by me.